django_cloud_tasks/signals.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from django.core.exceptions import ValidationError
+from django.db.models.signals import post_save, pre_save
+from django.dispatch import receiver
+
+from django_cloud_tasks import models, tasks
+
+
+@receiver(post_save, sender=models.Routine)
+def enqueue_routine_task_after_creation(sender, instance, created, **kwargs):
+    if created:
+        tasks.RoutineTask(routine_id=instance.pk)
+
+@receiver(pre_save, sender=models.Routine)
+def assure_valid_task_name(sender, instance, **kwargs):
+    try:
+        __import__(instance.task_name)
```

Comment:
Importing right now just to check might be expensive, you can just verify if the task is registered (like [here](https://github.com/flamingo-run/django-cloud-tasks/blob/main/django_cloud_tasks/views.py#L14)) and trust the rest of this lib has done its job.